cask.py

```diff
@@ -1683,5 +1683,19 @@ def __init__(self, *args, **kwargs):
 
 class Points(Object):
     """Points I/O Object subclass."""
+    _sample_class = alembic.AbcGeom.OPointsSchemaSample
     def __init__(self, *args, **kwargs):
         super(Points, self).__init__(*args, **kwargs)
+
+    def set_positions(self, *args):        
+        content = args[0]
+        # print content
+        positions = imath.V3fArray(len(content))
+        ids = imath.IntArray(len(content))
+        for i in range(0, len(content)):
```

Comment:
You can then enumerate on the positions to get the id,

```suggestion
        for i, position in enumerate(positions):
```